src/config/containers-and-vms/lxc.md

```diff
@@ -114,6 +119,40 @@ access to other containers. To isolate containers from each other, alter the
 create each container. Trying to fix permissions on a container created with the
 wrong map is possible, but inconvenient.
 
+## Incus
+
+Incus provides an alternative interface to LXC's `lxc-*` utilities. However, it
+does not require the configuration described in [the previous section](#lxc).
+
+Edit `/etc/rc.conf` and set the `CGROUP_MODE` variable to `unified`. Install the
+`incus` package, and [enable](../services/index.md#enabling-services) the
+`incus` service.
+
+Incus users must be belong to the `_incus-admin` group.
+
+Optionally enable some users to have limited acces to incus as described
+[here](https://linuxcontainers.org/incus/docs/main/howto/projects_confine/#confine-projects-to-specific-incus-users).
+Add users to the `_incus` group, and enable the `incus-user` service
+
+Note that `incus-user` will initialize the default incus profile when it is
+started. To avoid default configuration initialize incus for youself with `incus
+admin init` before enabling `incus-user`.
+
+**WARNING:** `incus-user` will also replace the networking config of the default
+profile if it appears invalid. runit does not have socket activation so this
+happens when when the machine boots instead of when a user attempts to access
+it. If the default profile uses a [bridge
+interface](../network/index.md#bridge-interfaces) `incus-user` may replace it on
+boot.
```

Comment:
```suggestion
> Warning: `incus-user` will also replace the networking config of the default
> profile if it appears invalid. runit does not have socket activation so this
> happens when when the machine boots instead of when a user attempts to access
> it. If the default profile uses a [bridge
> interface](../network/index.md#bridge-interfaces) `incus-user` may replace it on
> boot.
```